template/data/src/main/java/co/nimblehq/template/data/repository/RepositoryImpl.kt

```diff
@@ -15,8 +15,9 @@ class RepositoryImpl constructor(
         try {
             val result = apiService.getResponses().toModels()
             emit(result)
-        } catch (e: Exception) {
-            throw e
+        } catch (exception: Exception) {
+            // TODO do error mapping here
```

Comment:
Shall we start the comment with Capital?
```suggestion
            // TODO: Do error mapping here
```

Reply:
@lydiasama I think it's too strictly to follow that. I can't find any convention of Kotlin or Java requires to have `:` and start with a capital char. Let's make the dev's life easier 🙏 

https://www.jetbrains.com/help/idea/using-todo.html
https://developer.android.com/kotlin/style-guide#documentation
https://nimblehq.co/compass/development/code-conventions/kotlin/#comments

Reply:
Then could you please update[ this TODO comment](https://github.com/nimblehq/android-templates/pull/292/files#diff-eb44200d9d6b03ba7a180a699c76534b141ad10297fb217d311a8fa4733a5a9dR13) to align with others?

Reply:
@lydiasama It's not a big deal I think.
- It's not in my PR scope.
- We will remove it very soon.

Reply:
> We will remove it very soon.

@luongvo  When will we remove it? Did we have a plan to continue it already?
And also, I feel it's not neat to have all comments that don't align with the same pattern. 🤔 So instead of updating the new comments that you added, you can update the old one to be consistent.

Reply:
@lydiasama FYI I removed it here already https://github.com/nimblehq/android-templates/pull/311/files#diff-2ab311aa3921aeffb9dd9eb349a8ec774b2e63220b2793c49db434d68d0c52b2L19